src/sspi/negotiate/network_client.rs

```diff
@@ -1,7 +1,14 @@
+use std::fmt::Debug;
```

Comment:
I see you moved this module under `negotiate`. I understand this can be used at a few places (including kerberos) and it’s generic enough so I feel it could be at top level, what do you think?